### DISCUSSION TITLE
Openweathermap -> change unity m/s to km/h

### DISCUSSION THREAD

**Discussion by Frankastro85** [AllskyTeam/allsky-modules#98] [General]
Hello,
Just a simple question. I use the OpenweatherMap on my Allsky (http://www.villageduciel.fr/allsky/).
But for me, It's clearer to have the speed of wind in km/h and not m/s. I can calculate the speed (just multiply by 3.6) but is-it possible to had a formula in the variable ? Or change directly the unity ?
Thank you

**Comment by Alex-developer** [comment]
Apologies for the delay in replying !

I have created an issue for this and we will look at it after the next release.

**Reply by Frankastro85** [reply]
Thank you !
